vendor/oras.land/oras/internal/docker/mediatype.go

```diff
@@ -13,12 +13,9 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package slices
+package docker
 
-// Clone returns a shallow copy of the slice.
-func Clone[S ~[]E, E any](s S) S {
-	if s == nil {
-		return nil
-	}
-	return append(make(S, 0, len(s)), s...)
-}
+// docker media types
```

Comment:
what is this change?

Reply:
In order to import some of the packages (I was pairing with Adam) we had to run make vendor, otherwise it was throwing errors. So basically all the files changed under vendor were from the command running